keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -546,19 +550,7 @@ https://storage.googleapis.com/k8s-triage/index.html
 
 -->
 
-An important integration test for this migration will be a **loopback** test, exercising the
-WebSocket client and the StreamTranslator proxy. This test creates two test servers: a
-proxy server handling the stream translation, and a SPDY server which sends received data
-from one stream (e.g. stdin) back down another stream (e.g. stdout). This test will
-send random data from the WebSocket client to the StreamTranslator proxy, which then
-sends the data to the test SPDY server.
-
-WebSocket client  <->  Proxy Server (StreamTranslator)  <->  SPDY Server
-
-Once the data is received back at the WebSocket client on the separate stream, it
-is compared to the data that was sent to ensure the data is the same. This **loopback**
-test has been implemented in a proof-of-concept PR, validating the WebSocket client
-and the StreamTranslator proxy.
+- TBD
```

Comment:
I think it's better to indicate that unit tests and e2e tests provide sufficient coverage than to mark this as TBD

Reply:
Done.